content/intro-to-storybook/angular/en/simple-component.md

```diff
@@ -116,64 +117,38 @@ There are two basic levels of organization in Storybook: the component and its c
   - Story
   - Story
 
-To initiate Storybook we first call the `storiesOf()` function to register the component. We add a display name for the component –the name that appears on the sidebar in the Storybook app.
+To tell Storybook about the component we are documenting, we create a `default` export that contains:
+
+- `component` -- the component itself,
+- `title` -- how to refer to the component in the sidebar of the Storybook app,
+- `excludeStories` -- information required by the story, but should not be rendered by the Storybook app.
+
+To define our stories, we export a function for each of our test states to generate a story. The story is a function that returns a rendered element (i.e. a component class with a set of props) in a given state---exactly like a React [Stateless Functional Component](https://reactjs.org/docs/components-and-props.html).
```

Comment:
I would change the react link to the angular documentation : https://angular.io/guide/component-interaction

Reply:
@geromegrignon i left it temporarily. I was doing some research on this and it would seems that a better wording could be "....exactly like a [Stateless Functional Component](https://angular.io/guide/component-interaction), leaving out React out of the "picture". Do you agree?